lib/TPP/Dialect/Xsmm/XsmmUtils.cpp

```diff
@@ -8,18 +8,60 @@
 
 #include "TPP/Dialect/Xsmm/XsmmUtils.h"
 #include "TPP/Dialect/Xsmm/XsmmOps.h"
+#include "TPP/Transforms/Utils/VNNIUtils.h"
 #include "TPP/Transforms/Utils/ValueUtils.h"
 #include "mlir/Dialect/Func/IR/FuncOps.h"
+#include "mlir/Dialect/LLVMIR/LLVMDialect.h"
```

Comment:
I'm surprised by the need of the LLVM dialect header here.

Reply:
It's for memref expansion into llvm ops.